ui/app/src/services/audit.ts

```diff
@@ -61,6 +61,6 @@ export function fetchAuditLog(options: AuditOptions): Observable<PagedArray<Audi
 export function fetchAuditLogEntry(id: number, siteId?: string): Observable<AuditLogEntry> {
   const qs = toQueryString({ siteId }, { skipNull: true });
   return get<Api2ResponseFormat<{ auditLog: AuditLogEntry }>>(`/studio/api/2/audit/${id}${qs}`).pipe(
-    pluck('response', 'auditLog')
+    map(({ response: { auditLog } }) => auditLog)
```

Comment:
Avoid this notation. Use the raw argument on the left with optional chaining on the right side.